config/locales/de.yml

```diff
@@ -75,7 +75,7 @@ de:
         title: "Ausleihanfrage: %{item}"
         description: "%{receiver} möchte %{item} von dir ausleihen."
         description_accepted: "%{receiver} hat %{item} von dir ausgeliehen."
-        description_declined: "Du hast abgelehnt %{item} an %{receiver} auszuleihen"
+        description_declined: "Du hast abgelehnt, %{item} an %{receiver} auszuleihen."
```

Comment:
```suggestion
        description_declined: "Du hast es abgelehnt, %{item} an %{receiver} auszuleihen."
```

Reply:
Finde ich ehrlich gesagt schlechter.

Reply:
@dasGoogle Are you fine with this change?

Reply:
Wie gesagt, ich finde, es klingt schlechter. Wenn ich es also zu entscheiden habe, würde ich die Änderung ablehnen. Wenn @benn02 es so will, dann ist es so.

Reply:
Aber ehrlicherweise ist mir das auch relativ egal

Reply:
<ich habe DeepL gefragt ... weil ich mir nicht sicher war, aber es sind, glaube ich beide Sachen richtig. "Du hast abgelehnt, A an B auszuleihen" ist glaube ich umgangssprachlicher, deshalb die Änderung